app/Domains/Vault/ManageVault/Api/Controllers/VaultController.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+namespace App\Domains\Vault\ManageVault\Api\Controllers;
+
+use App\Domains\Vault\ManageVault\Services\CreateVault;
+use App\Domains\Vault\ManageVault\Services\DestroyVault;
+use App\Domains\Vault\ManageVault\Services\UpdateVault;
+use App\Http\Controllers\ApiController;
+use App\Http\Resources\VaultResource;
+use App\Models\Vault;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+use Illuminate\Database\QueryException;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+
```

Comment:
Below changes requires these uses:
```suggestion
use Knuckles\Scribe\Attributes\BodyParam;
use Knuckles\Scribe\Attributes\QueryParam;
use Knuckles\Scribe\Attributes\ResponseFromApiResource;
use Knuckles\Scribe\Attributes\Response;

```